inductiva/structures/simulators/fenicsx.py

```diff
@@ -0,0 +1,44 @@
+"""FEniCSx module of the API for finite element analysiss."""
+
+from typing import Optional, List
+from uuid import UUID
+
+from inductiva import types, tasks
```

Comment:
Try to follow Inductiva's guideline on imports.
Check Hugo's email.
It goes along these lines:

https://google.github.io/styleguide/pyguide.html#22-imports

Screenshot 2023-08-01 at 12.48.53.png

So, according to the guidelines this is NOT RECOMMENDED:

Screenshot 2023-08-01 at 12.53.08.png
And the RECOMMENDED version is below:

Screenshot 2023-08-01 at 12.51.24.png


Note: other different Python guidelines are ok with importing classes, but not the one we are following, so let's stick to it.

Reply:
Thanks Luis, for the input! I will make the necessary changes to adhere the Google style and Hugo's recommendations